pkg/test/framework/components/istio/operator.go

```diff
@@ -177,6 +178,8 @@ func (i *operatorComponent) Close() error {
 		scopes.Framework.Infof("=== SUCCEEDED: Cleanup Istio in %v [Suite=%s] ===", delta, i.ctx.Settings().TestID)
 	}()
 
+	i.dumpGeneratedManifests()
```

Comment:
Seems like its missing a check for `DumpKubernetesManifests`?

Reply:
thanks!